src/applications/disability-benefits/all-claims/tests/schema/minimal-test.json

```diff
@@ -16,47 +16,24 @@
       "zipCode": "19234"
     },
     "view:contactInfoDescription": {},
-    "view:vaMedicalRecordsIntro": {},
-    "vaTreatmentFacilities": [
```

Comment:
Should this be removed? Going through the form flow, they have to choose _some_ type of evidence, right?

Reply:
Nope, they can elect to choose no evidence.